readme.md

```diff
@@ -113,19 +113,236 @@ can be found in our Kubernetes configuration https://github.com/zalando-incubato
+        metadata:
+          name: skipper-ingress
+          labels:
+            application: skipper-ingress
+            version: v0.9.115
+            component: ingress
+          annotations:
+            scheduler.alpha.kubernetes.io/critical-pod: ''
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: master
+                    operator: DoesNotExist
+          tolerations:
+          - key: CriticalAddonsOnly
+            operator: Exists
+          hostNetwork: true
+          containers:
+          - name: skipper-ingress
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.9.115
+            ports:
+            - name: ingress-port
+              containerPort: 9999
+              hostPort: 9999
+            args:
+              - "skipper"
+              - "-kubernetes"
+              - "-kubernetes-in-cluster"
+              - "-address=:9999"
+              - "-proxy-preserve-host"
+              - "-serve-host-metrics"
+              - "-enable-ratelimits"
+              - "-experimental-upgrade"
+              - "-metrics-exp-decay-sample"
+            resources:
+              limits:
+                cpu: 200m
+                memory: 200Mi
+              requests:
+                cpu: 25m
+                memory: 25Mi
+            readinessProbe:
+              httpGet:
+                path: /kube-system/healthz
+                port: 9999
+              initialDelaySeconds: 5
+              timeoutSeconds: 5
+
+
+We now deploy a simple demo application serving html:
+
+    % cat demo-deployment.yaml
+    apiVersion: apps/v1beta1
+    kind: Deployment
+    metadata:
+      name: skipper-demo
+    spec:
+      replicas: 2
+      template:
+        metadata:
+          labels:
+            application: skipper-demo
+        spec:
+          containers:
+          - name: skipper-demo
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.9.117
+            args:
+              - "skipper"
+              - "-inline-routes"
+              - "* -> inlineContent(\"<body style='color: white; background-color: green;'><h1>Hello!</h1>\") -> <shunt>"
+            ports:
+            - containerPort: 9090
+
+We deploy a service type ClusterIP that we will select from ingress:
+
+    % cat demo-svc.yaml
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: sszuecs-demo
+      labels:
+        application: skipper-demo
+    spec:
+      type: ClusterIP
+      ports:
+        - port: 80
+          protocol: TCP
+          targetPort: 9090
+          name: external
+      selector:
+        application: sszuecs-demo
+
+To deploy both, you have to run:
+
+    % kubectl create -f demo-deployment.yaml
+    % kubectl create -f demo-svc.yaml
+
+Now we have a skipper-ingress as daemonset running exposing port 9999
```

Comment:
Now we have a skipper-ingress _running_ as a daemonset _and_ exposing _the_ port 9999
on each worker node, a backend application _running_ with 2 replicas
that serves some html _on_ port 9090, and we expose a cluster service
on port 80. Besides skipper-ingress, the deployment and the service can not
be reached from outside the cluster. _Now we_ expose _the service_ with Ingress
to _the external_ network: